Sources/BraintreeVenmo/Models/VenmoCreatePaymentContextGraphQLBody.swift

```diff
@@ -111,18 +89,19 @@ struct VenmoCreatePaymentContextGraphQLBody: BTGraphQLEncodableBody {
                     var taxAmount: String?
                     var shippingAmount: String?
                     var totalAmount: String?
-                    let lineItems: [LineItem]?
+                    var lineItems: [LineItem]?
                     
-                    func encode(to encoder: Encoder) throws {
-                        var container = encoder.container(keyedBy: CodingKeys.self)
+                    init(request: BTVenmoRequest) {
+                        self.subTotalAmount = request.subTotalAmount
+                        self.discountAmount = request.discountAmount
+                        self.taxAmount = request.taxAmount
+                        self.shippingAmount = request.shippingAmount
+                        self.totalAmount = request.totalAmount
                         
-                        try container.encodeIfPresent(subTotalAmount, forKey: .subTotalAmount)
-                        try container.encodeIfPresent(discountAmount, forKey: .discountAmount)
-                        try container.encodeIfPresent(taxAmount, forKey: .taxAmount)
-                        try container.encodeIfPresent(shippingAmount, forKey: .shippingAmount)
-                        try container.encodeIfPresent(totalAmount, forKey: .totalAmount)
-                        if let lineItems, !lineItems.isEmpty {
-                            try container.encode(lineItems, forKey: .lineItems)
+                        if let lineItems = request.lineItems, !lineItems.isEmpty {
+                            self.lineItems = lineItems.compactMap {
+                                LineItem(item: $0)
+                            }
```

Comment:
nit, I think we can single line this:
```suggestion
                            self.lineItems = lineItems.compactMap { LineItem(item: $0) }
```